### DISCUSSION TITLE
Chia Mojo not recieved

### DISCUSSION THREAD

**Discussion by hj-crypto** [Chia-Network/chia-blockchain#9476] [Support]
Hi Team,

I want to start Chia minig . I order to do this . I need to join Chia pool .For this I need Chia Mojo but I didn't recieved yet. I device is synced from last 2 days . Kindly Help.

Thanks in Advance.

**Comment by marssystems** [comment]
Go to this faucet and get yourself some chia -

https://chia2start.eu/faucet.html
